core/src/main/scala/cats/arrow/NaturalTransformation.scala

```diff
@@ -1,6 +1,23 @@
 package cats
 package arrow
 
-trait NaturalTransformation[F[_], G[_]] extends Serializable {
+trait NaturalTransformation[F[_], G[_]] extends Serializable { outer =>
   def apply[A](fa: F[A]): G[A]
+
+  def compose[E[_]](f: NaturalTransformation[E, F]): NaturalTransformation[E, G] =
+    new NaturalTransformation[E, G] {
+      def apply[A](fa: E[A]): G[A] = outer.apply(f(fa))
+    }
+
+  def andThen[H[_]](f: NaturalTransformation[G, H]): NaturalTransformation[F, H] =
+    f.compose(outer)
+
+  def function[A]: F[A] => G[A] = fa => apply(fa)
```

Comment:
true but it would be good to be consistent, we could have the same discussion regarding parameter lists. For example:
```scala
def foo(a: A)(b: B): C
def foo2(a: A): B => C
def foo3: A => B => C
```